### DISCUSSION TITLE
Domotica - misura energia

### DISCUSSION THREAD

**Discussion by gianry** [indomus/forum#2195] [Hardware & Firmware]
Salve,
ho un abitazione in cui vorrei monitorare il consumo energetico generale , avere lo storico consumi, visualizzare l'assorbimento istantaneo e gestire il carico di 4- 5 prese per evitare che superiori una certa soglia di potenza.
Tutto cio' lo potrei fare acuqistando i prodotti Bticino living now. Qualora volessi invece utilizzare home assistant e programmarlo quale hardware potrei acquistare ? Per esempio posso acquistare Bticino   fc80gcs come sonda toroidale e Bticino 4431c come rele per presa ?

avrei bisogno di un gateway ? posso contare su di un NAS Qnap con cpu celeron e mi sembra che li' si possa installare home assistant.

grazie mille per l'aiuto.

**Comment by KC-inDomus** [comment]
> Salve, ho un abitazione in cui vorrei monitorare il consumo energetico generale , avere lo storico consumi, visualizzare l'assorbimento istantaneo e gestire il carico di 4- 5 prese per evitare che superiori una certa soglia di potenza. Tutto cio' lo potrei fare acuqistando i prodotti Bticino living now. Qualora volessi invece utilizzare home assistant e programmarlo quale hardware potrei acquistare ? Per esempio posso acquistare Bticino fc80gcs come sonda toroidale e Bticino 4431c come rele per presa ?
> 
> avrei bisogno di un gateway ? posso contare su di un NAS Qnap con cpu celeron e mi sembra che li' si possa installare home assistant.
> 
> grazie mille per l'aiuto.

A monte, metti questo: https://indomus.it/componenti/shelly-em/
Per installare HA hai molte alternative (tra le quali QNAP): https://indomus.it/formazione/primi-passi-con-home-assistant-da-dove-cominciare/

**Reply by gianry** [reply]
e come gestisco l'interruzione di energia sulle singole prese ? il gateway chi lo fà ?
